WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -108,13 +108,6 @@ public void onSaveInstanceState(Bundle outState) {
     }
 
     @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        // retain this fragment across configuration changes
-        setRetainInstance(true);
```

Comment:
Removing the `setRetainInstance` has the side effect of the SEND button never getting re-enabled when the network calls finish. I guess the SEND button enabling/disabling should be removed as well, BUT: that would actually "cannibalise" this PR.

At this point, and basically aligning with the Slack chat we had, I think this PR should be left with `setRetainInstance` in and be allowed to continue the work for a better UX here and also fork the #4332 fix into a separate PR. WDYT?